README.md

```diff
@@ -1,24 +1,59 @@
-# 👾 Processbot
+# 👾 processbot
 
 # Commands
 
 To be posted as a comment in the Pull Request. **Your whole comment should only have the command**.
 
-- `bot merge` to automatically merge it once checks pass (if approvals have been
-  given).
-- `bot merge force` to attempt merge without waiting for checks (if approvals
-  have been given).
-- `bot merge cancel` to cancel a pending `bot merge`.
-- `bot compare substrate` to see a diff between current branch's Substrate.
+- `bot merge`: [if approved](#criteria-for-merge), merge once checks pass.
+- `bot merge force`: [if approved](#criteria-for-merge), merge immediately
+  while disregarding checks.
+- `bot merge cancel`: cancel a pending `bot merge`.
+- `bot compare substrate`: see a diff between current branch's Substrate
   version and the latest Polkadot release's Substrate version.
-- `bot rebase` to merge origin/master.
-- `bot burnin` to build and deploy the PR for a burn-in test.
+- `bot rebase`: merge origin/master into the PR.
+- `bot burnin`: build and deploy the PR for a burn-in test.
 
-Commands should all be defined in [constants.rs](./src/constants.rs).
+Note: The commands will only work if you are a member of the organization where
+this bot is installed. Organization membership is always gotten fresh from the
+the Github API at the time a comment arrives.
 
-# Deployment
+## Relation to CI
+
+processbot categorizes CI statuses as following, ranked in descending order of importance:
+
+### 1. Required
+
+Criteria:
+
+- Required through Github settings
+
+They are meant to be blockers so can't be skipped anyhow.
+
+### 2. Important
+
+Criteria:
 
-Note: As of March 26 of 2021, the bot has been running on production with [tag 0.5.0](https://github.com/paritytech/parity-processbot/releases/tag/v0.5.0) since October 27 of 2020. Although many bugs have been reported since then, that is the current "stable" version which developers are used to. `master` has since then received commits which have not landed in production; therefore, if a "doom scenario" comes up, it's advised that you always deploy that tag instead of retagging master for the time being.
+- Derived from Gitlab Jobs which **do not** have `allow_failure: true`
+- Not required through Github settings
+
+They are relevant but not blockers, thus can be skipped with `bot merge force`
+but will not pass `bot merge`. Note that the companion build system follows the
+logic of `bot merge`, thus a brittle job in this category might get in the way
+of a companion merge.
+
+### 3. Fallible
+
+Criteria:
+
+- Derived from Gitlab Jobs which have `allow_failure: true`
+- Not required through Github settings
```

Comment:
```suggestion
```
It's required anyway :)

Reply:
Makes sense. I've reformatted this section for brevity and omitted this.